Modules/@babylonjs/react-native-basekit/scripts/install.js

```diff
@@ -0,0 +1,227 @@
+const https = require("https");
+const zlib = require("zlib");
+const tar = require("tar");
+const path = require("path");
+const fs = require("fs");
+
+function getCachePathCandidates() {
+  return [
+    process.env.npm_config_babylon_binary_cache,
+    process.env.npm_config_cache,
+  ].filter(function(_) {
+    return _;
+  });
+}
+
+function getCached(package, binaryFilename) {
+  return [
+    getCachePathCandidates(),
+    "BabylonReactNative",
+    package.version,
+    binaryFilename,
+  ].join("/");
+}
+
+function useCachedFile(filePath) {
+  return new Promise((resolve, reject) => {
+    if (fs.existsSync(filePath)) {
+      const gunzip = zlib.createGunzip();
+      const untar = tar.extract({ cwd: process.cwd() }); // Extract to the current working directory
+
+      const sourceStream = fs.createReadStream(filePath);
+      sourceStream
+        .pipe(gunzip) // Decompress the .tar.gz file
+        .pipe(untar) // Extract the tar contents to the cwd
+        .on("error", (err) => {
+          reject(err);
+        })
+        .on("end", () => {
+          resolve(true);
+        });
+    } else {
+      resolve(false);
+    }
+  });
+}
+
+function getArgument(name) {
+  const flags = process.argv.slice(2),
+    index = flags.lastIndexOf(name);
+
+  if (index === -1 || index + 1 >= flags.length) {
+    return null;
+  }
+
+  return flags[index + 1];
+}
+
+function getBinaryUrl(package, binaryFilename) {
+  var site =
+    getArgument("--brn-binary-site") ||
+    process.env.BRN_BINARY_SITE ||
+    process.env.npm_config_brn_binary_site ||
+    (package.nodeBabylonConfig && package.nodeBabylonConfig.binarySite) ||
+    "https://github.com/BabylonJS/BabylonReactNative/releases/download";
+
+  return [site, package.version, binaryFilename].join("/");
+}
+
+function downloadExtractAndCache(url, cachedFilePath, cb) {
+  const options = {
+    followRedirects: true, // Follow HTTP 3xx redirects
+  };
+
+  https
+    .get(url, options, (response) => {
```

Comment:
```suggestion
  https.get(url, options, (response) => {
```

Reply:
That's Prettier way to format this line. It's weird indeed.